src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer<E> implements MachineProducer<E> {
```

Comment:
Parametrization is not used, please see common mistakes (https://mate-academy.github.io/jv-program-common-mistakes/java-core/pecs/pecs.html#do-not-use-raw-types-always-parametrize-interfaces-and-classes-if-it-is-possible) This class should work only with excavator